src/chrome/content/rules/LastPass.xml

```diff
@@ -1,11 +1,8 @@
-<ruleset name="LastPass.com" default_off="Breaks addon ( https://www.eff.org/r.3Dq )">
+<ruleset name="LastPass.com">
   <target host="lastpass.com" />
   <target host="*.lastpass.com" />
 
   <securecookie host="^(?:.+\.)?lastpass\.com$" name=".*"/>
 
-  <rule from="^http://(?:www\.)?lastpass\.com/" to="https://lastpass.com/"/>
-
-	<rule from="^http://forums\.lastpass\.com/"
-		to="https://forums.lastpass.com/" />
+  <rule from="^http:" to="https:" />
```

Comment:
The old version of this rule rewrote only forums and www, but the new version will rewrite all subdomains. I suggest you should probably keep the old semantics unless you have a very strong reason not to.

That means changing the target host = "*.lastpass.com" to target host = "forums.lastpass.com", target host = "www.lastpass.com".